lesson02/assignment/css/_sizes.scss

```diff
@@ -0,0 +1,20 @@
+// Borders
+$extra-thin: .5rem solid;
+$thin-border: 1rem solid;
+$medium-border: 1.5rem solid;
+$large-border: 2.5rem solid;
+
+// rems
+$size-rem-one: 0.6rem;
+$size-rem-two: 0.7rem;
+$size-rem-three: 0.8rem;
+$size-rem-four: 0.9rem;
+$size-rem-five: 1rem;
+$size-rem-six: 1.5rem;
+$size-rem-seven: 2rem;

```

Comment:
It might be nice to use more semantic names for these, like $font-extra-small, $font-small, etc. But this also works.

Reply:
I thought about it but didn't have any improvisation for naming custom variables. Fortunately, you gave me a hint. Thanks